docs/topics/db/sql.txt

```diff
@@ -54,6 +54,23 @@ Of course, this example isn't very exciting -- it's exactly the same as
 running ``Person.objects.all()``. However, ``raw()`` has a bunch of other
 options that make it very powerful.
 
+
+.. versionadded:: 2.1
```

Comment:
The `.. versionadded:: 2.1` annotation should be self-contained as described in [Documenting new features](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features).

Reply:
This block is still not self contained, so that it can be removed after 2 versions.